### PR TITLE
Eng 982

### PR DESCRIPTION
**Pull request by egh**
JIRA issue: https://jira.plos.org/jira/browse/ENG-892
- Do not escape invitation body
- Output the string `false` when outputting false stuff
- Also export the latest version of a card (as well as snapshot)
- Export cards where the user didn't fill out anything (this is useful info)
- Add basic export of card content

### CODE REVIEW

File: app/views/export/_answer.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div class="answer">
+  <%= answer.try(:value).try(:to_s).try(:html_safe) || "[No response]" %>
```
Comment (sgatchell) [inline app/views/export/_answer.html.erb:2]: I'm just wasting time pointing out that this version of ruby supports the safe navigation operator `&.` and noting that It's not worth the effort to update Aperta to use it.

**Review by sgatchell — APPROVED** [review]
:shipit:

**Comment by sgatchell** [comment]
oh no big deal but jira says this about the APERTA issue: `It may have been deleted or you don't have permission to view it. `

File: app/views/export/_answer.html.erb (hunk 1 above)
Comment (egh) [reply app/views/export/_answer.html.erb:2]: whoa, that's awesome. I didn't know!

Comment (sgatchell) [reply app/views/export/_answer.html.erb:2]: Well then I'm happy to have mentioned it!

**Pull request merged**